app/src/test/java/com/codesoom/assignment/ProductNotFoundExceptionTest.java

```diff
@@ -0,0 +1,22 @@
+package com.codesoom.assignment;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class ProductNotFoundExceptionTest {
+
+    @Test
+    @DisplayName("없는 아이디를 찾을때는 에러가 발생한다.")
+    void productNotFoundException() {
+        Long givenNotExistedId = 100L;
+
+        ProductNotFoundException productNotFoundException
+                = new ProductNotFoundException(givenNotExistedId);
+
+        assertThat(productNotFoundException.getMessage())
+                .isEqualTo("Product not found: " + givenNotExistedId);
```

Comment:
너무 변수가 길어서 이해하기 어려운 것 같습니다. 변수의 수명이 짧다면 단순한 이름을 사용해도 괜찮습니다.
```suggestion
        ProductNotFoundException exception
                = new ProductNotFoundException(givenNotExistedId);

        assertThat(exception.getMessage())
                .isEqualTo("Product not found: " + givenNotExistedId);
```